packages/react-scripts/scripts/utils/verifyTypeScriptSetup.js

```diff
@@ -18,7 +18,8 @@ const immer = require('react-dev-utils/immer').produce;
 const globby = require('react-dev-utils/globby').sync;
 
 function writeJson(fileName, object) {
-  fs.writeFileSync(fileName, JSON.stringify(object, null, 2) + os.EOL);
+  // JSON.stringify by contract produces \n always
```

Comment:
I don't think we need this comment. Otherwise it looks good.

Reply:
👍 Done